leftwm-core/src/utils/child_process.rs

```diff
@@ -87,6 +88,31 @@ impl Nanny {
     /// Could be caused by inadequate permissions.
     pub fn run_global_up_script() -> Result<Option<Child>> {
```

Comment:
Is it by the way planned to have the multiple-up-script-feature only in the main directory?

Reply:
Yes, as described in #547

Reply:
hm... ok